pages/am/3.x/breaking-changes/breaking-changes.adoc

```diff
@@ -6,6 +6,7 @@
 
 This section outlines the changes you should consider when migrating your application from one version of Gravitee.io Access Management to another.
 
+* link:{{ '/am/current/am_breaking_changes_3.20.html' | relative_url }}[Breaking changes in 3.20]
```

Comment:
This link should also be added to `_data/sidebars/am_3_x_sidebar.yml`

Reply:
added the missing link